### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by twallnerWaretec**
reworked db schema for multilingual support

### CODE REVIEW

**Comment by twallnerWaretec** [comment]
@Frando the cba unit test fails because in the fixture file the pubdate has a different timezone than the newly fetched one in the test.
What can we do here?

**Comment by Frando** [comment]
> @Frando the cba unit test fails because in the fixture file the pubdate has a different timezone than the newly fetched one in the test. What can we do here?

Sorry for late reply, missed the notification somehow.
You can try to recreate the fixtures by doing  `export CREATE_FIXTUES=1` before running `yarn test`, and then commiting the changed fixtures. See https://github.com/repco-org/repco/blob/main/docs/Development/testing.md#tests-with-fixture for details.

**Comment by Frando** [comment]
@twallnerWaretec tests are failing, did you investigate why this is the case?

**Comment by Frando** [comment]
We tried to review this, and found two immediate spots:

* Basic tests are not passing, it seems they are not updated at all for the language changes yet?
* `ContentItem.subtitle` is still a string, not an object - should this not be translatable as well?

**Comment by Frando** [comment]
Hi @twallnerWaretec can you post an update on where this branch stands? We really should move forward with merging this in main.

* This branch does not include recent updates on main. #104 merges the updates from main into this branch (including the improved error handling)
* Test do not pass. #104 makes some of the tests pass, but not yet all.

Are you going to merge #104 into this branch? And are you going to update the remaining tests?

We are tasked with developing new functionality (fetching subtitles), and I'd prefer to develop this on top of the translations branch right away to save mergeconflict time. However we really should not do features-on-top-of-featurebranches much, it makes everything more complicated.  So I'd prefer to get this branch done and merged rather sooner than later.

@twallnerWaretec let me know what you think and what you need to finish this.

File: packages/repco-core/src/datasources/activitypub.ts [hunk 1]
```diff
@@ -591,9 +611,14 @@ export class ActivityPubDataSource
       subtitleEntities && entities.push(...subtitleEntities)
     }
 
+    var titleJson: { [k: string]: any } = {}
+    titleJson[video.language?.name || 'de'] = { value: video.name }
```
Comment (Frando) [inline packages/repco-core/src/datasources/activitypub.ts:615]: - Ingo reported a bug about `English` appearing as an object key, where it should be `en`. I suspect this line being the cause.

**Pull request merged**